srcpkgs/boost/template

```diff
@@ -112,3 +111,283 @@ boost-devel_package() {
 		vmove "usr/lib/*.so"
 	}
 }
+
+libboost_atomic-169_package() {
```

Comment:
there is no dash between the soname version and the package

libhunspell1.7
libcrypto44
libssl46
libtls18

Reply:
It will get otherwise ugly e.g. with python3169.

Reply:
that will make it different from everyone else, maybe do it only where it doesn't end with a number.

Reply:
Ok will do that.